query/iam/iam_subscription_owner_more_than_1.sql

```diff
@@ -0,0 +1,32 @@
+with owner_roles as (
+  select
+    role_name,
+    role_type, name,
+    title,
+    subscription_id
+  from
+    azure_role_definition
+  where
+    role_name = 'Owner'
+)
+select
+  -- Required columns
+  owner.subscription_id as resource,
+  case
+    when count(*) > 1 then 'ok'
+    else 'alarm'
+  end as status,
+    case
+    when count(*) = 1 then 'There is 1 owner.'
```

Comment:
There is 1 owner. > Only 1 owner associated.

Reply:
updated the reason as per the suggestion.